packages/runtime/container-runtime/src/test/types/validateContainerRuntimePrevious.generated.ts

```diff
@@ -591,24 +592,6 @@ declare type old_as_current_for_Interface_ISummaryConfigurationHeuristics = requ
  */
 declare type current_as_old_for_Interface_ISummaryConfigurationHeuristics = requireAssignableTo<TypeOnly<current.ISummaryConfigurationHeuristics>, TypeOnly<old.ISummaryConfigurationHeuristics>>
 
-/*
- * Validate forward compatibility by using the old type in place of the current type.
- * If this test starts failing, it indicates a change that is not forward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "Interface_ISummaryConfigurationWithSummaryOnRequest": {"forwardCompat": false}
- */
-declare type old_as_current_for_Interface_ISummaryConfigurationWithSummaryOnRequest = requireAssignableTo<TypeOnly<old.ISummaryConfigurationWithSummaryOnRequest>, TypeOnly<current.ISummaryConfigurationWithSummaryOnRequest>>
```

Comment:
This may be unrelated. I got this too in one of my PRs but went away on rebuild I believe.